src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/excel/MultipleMidEventValidationServiceTest.java

```diff
@@ -41,14 +41,12 @@ public void setUp() {
     @Test
     public void multipleValidationLogicCurrentMultiple() {
 
```

Comment:
This file has lines added at the start and end of each method and the class, I'd much rather the lines where used to breakup blocks of code in each method.

Reply:
removed lines